README.md

```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
 * [Implement customized TrainingService](docs/en_US/TrainingService/HowToImplementTrainingService.md)
 
 
-## **External Repositories**
+## **External Repositories and References**
 Now we have some external usage examples run in NNI from our contributors. Thanks our lovely contributors. And welcome more and more people to join us!
+
+
 * Run [ENAS](examples/tuners/enas_nni/README.md) in NNI
 * Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) in NNI 
-* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI
+* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI <br>
```

Comment:
How about adding recommenders with this link https://github.com/microsoft/recommenders/blob/master/notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb ?

Reply:
Ok,fixed , but... do we need permission from the author?

Reply:
ok fixed , but.. do we need permission from the author ?